desktop/cmp/rest/RestGridModel.js

```diff
@@ -57,18 +80,42 @@ export class RestGridModel {
     constructor({
         actionEnabled,
         actionWarning,
+        actionColEnabled,
         unit = 'record',
         filterFields,
         enhanceToolbar,
         editors = [],
+        columns,
         ...rest
     }) {
         this.actionEnabled = Object.assign(this.actionEnabled, actionEnabled);
         this.actionWarning = Object.assign(this.actionWarning, actionWarning);
+        this.actionColEnabled = actionColEnabled;
```

Comment:
If we exported the actions separately, would it be trivial matter for the application to simply specify a column of the form `{...actionsCol, actions: [recordEditAction, recordDeleteAction]}`.

Advantage here is that the application has flexibility to place and size the column, but also mix in its own actions in to the same column.

RestGridModel could even, say,  make its actions toolbar a public property, which could be enhanced along the same lines.

This generally suggests a great pattern for any API -- provide Action configs as public exports which can then be fed back to Grids as applications see fit.  Very similar to our column API.
Could be much more extensible than the 'token' based approach we were using.

Reply:
If we don't want to do my suggested change now (which I totally understand).  I do think we should remove this property 'actionColEnabled' from RestGrid.  I don't think its the right API, and we have no immediate need for it.   We can take some more time to think about the best pattern for exposing actions in this and other APIs.